package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "sqs-consumer",
-  "version": "3.8.0",
```

Comment:
should this be a major version?

Reply:
wasn't going to do release just yet but can do